gems/activerecord/6.0/activerecord-6.0.rbs

```diff
@@ -0,0 +1,5 @@
+module ActiveRecord
+  module Enum
+    def enum: (Hash[Symbol, (Hash[(Symbol | String), untyped] | Array[Symbol | String] | bool)] definitions) -> void
```

Comment:
ActiveRecord::Enum#enum method in 6.0 version
https://github.com/rails/rails/blob/28bb76d3efc39b2ef663dfe2346f7c2621343cd6/activerecord/lib/active_record/enum.rb#L150